modules/ROOT/pages/scopes-and-collections-config.adoc

```diff
@@ -91,138 +84,94 @@ This example covers a configuration with 1 custom Scope and 2 custom Collections
+For more information, see xref:import-filter.adoc[Import Filters].
+|                       | The import filter checks if the `type` field of the document is `mobile`. If it is, the document will be imported; otherwise, it will be skipped.
+|                       |
+| collection2           | The name of the second collection within `scope1`.
+| sync                  | The sync function associated with `collection2`. This function is triggered when a document is created or updated in the collection. 
+For more information, see xref:sync-function.adoc[Sync Functions].
+|                       | The sync function sends a message to the `collection1` channel.
+| import_filter         | The import filter function associated with `collection2`. This function is used when importing documents into the collection. 
+For more information, see xref:import-filter.adoc[Import Filters].
+|                       | The import filter checks if the `type` field of the document is `mobile`. If it is, the document will be imported; otherwise, it will be skipped.
+|                       |
+| num_index_replicas    | The number of index replicas. In this case, it is set to 0, meaning no replicas of indexes will be created for this database.
+|===
+
+== Configuration Example: Dynamic Access Grant
 
-=== Sync Function:
 [source,javascript]
 ----
 function(doc, oldDoc, meta) {
-    if (doc.type != "default") {
-        throw({forbidden : "Rejected document"});
-    }
-    channel("legacy");
+   if (doc.userId) {
+       channelId = "channel.profile:"+ doc.userId;
+       // Assign document to channel
+       channel(channelId);
+       // Grant user access to channel
+       access(doc.userId, channelId);
+   }
+   if (doc.providerId) {
+       // Grant provider user access to channel
+       access(doc.providerId, channelId);
+   }
 }
 ----
 
-Whenever a document is synced or updated in collection1, a function called `sync` is executed. 
-This function has three parameters:
+|===
+| Field           | Explanation
 
-* New document (doc)
-* Existing document (oldDoc)
-* Document metadata (meta)
+| doc             | The `doc` object contains the data for the current document being processed.
+| oldDoc          | The `oldDoc` object contains the previous version of the document (if available).
+| meta            | The `meta` object contains metadata about the document, such as its expiration time.
+| doc.userId      | Checks if the `doc` object contains a field called `userId`.
+| channelId       | The string that represents the channel where the document is assigned.
+| channel()       | A function used to assign the document to a channel represented by the `channelId`.
+| access()        | A function used to grant access to a user for a specific channel.
+| doc.providerId  | Checks if the `doc` object contains a field called `providerId`.
```

Comment:
This table is nice and compact, so OK to leave it formatted like this 👍 
(but if it gets bigger, then move to new lines like the other one)